domain/pom.xml

```diff
@@ -8,12 +8,18 @@
         <version>1.0-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
-
-    <artifactId>domain</artifactId>
-
     <properties>
         <maven.compiler.source>8</maven.compiler.source>
         <maven.compiler.target>8</maven.compiler.target>
     </properties>
 
+    <artifactId>domain</artifactId>
+    <dependencies>
+        <dependency>
+            <groupId>com.issoft.training</groupId>
+            <artifactId>stores</artifactId>
```

Comment:
you shoud put all domain models to that module, and it should not depend on any others